jsontableschema/types/geopoint.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import json
+import decimal
+from future.utils import raise_with_traceback
+from .. import exceptions
+from .. import utilities
+from .. import compat
+from . import base
+
+
+# Module API
+
+# TODO: review
```

Comment:
@roll please get all TODOs out into issues and not in code base: granted many were here before you touched the code.